pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -495,6 +513,79 @@ func (o *MustGatherOptions) Run() error {
 	return errors.NewAggregate(errs)
 }
 
+// runWorker processes Pods created for collecting each must-gather
+func (o *MustGatherOptions) runWorker(ns string, podChan chan *corev1.Pod, errCh chan error, wg *sync.WaitGroup) {
+	for pod := range podChan {
+		o.processNextWorkItem(ns, pod, errCh)
```

Comment:
Better to return error from `o.processNextWorkItem` and add it to `errCh` here.

Reply:
Done.